### ISSUE TITLE
Site based name - Adress already in use

### ISSUE THREAD

**Issue by lu6fer** [OpenMediaVault-Plugin-Developers/openmediavault-nginx#26]
The plugin seems to use old listen nginx syntax, witch cause error in omv 4.x

Juste drop the [ssl_]listen_ipv4 from the mkconf will fix this.

**Comment by scandinave** [comment]
I have the same problem with omv v3. The plugin write 
`listen 443 ssl;
 listen [::]:443 ssl;
`

Remove the first line make it works

**Comment by lu6fer** [comment]
I don't know why @ryecoaaron has reverted his change about this.
But for me it's the only solution to have site-based name on omv-nginx plugin.

**Comment by ryecoaaron** [comment]
Because the fix broke more installs than it fixed. I need to come up with a better fix.

**Comment by tjjh89017** [comment]
Not only OMV 4.x but also 3.x

BTW why not just listen for both IPv4 and IPv6?
https://github.com/tjjh89017/openmediavault-nginx/commit/88e7a905bec057dddc5886b066e14211e92abd52

**Comment by ryecoaaron** [comment]
That is the fix I reverted.  This breaks systems that are only configured for ipv4.

**Comment by scandinave** [comment]
My system is only configured with ipv4 and does not work. Is there any advance on this issues?

**Comment by ryecoaaron** [comment]
No, there has been no work done on this.  If you add a static ipv6 address, you shouldn't have the issue until it can be fixed.

**Comment by thanosasimo** [comment]
I found this on stack overflow.
Maybe it helps.
`Make sure you dont use old syntax:

server {
    listen :80;
    listen [::]:80;
}
The above syntax will cause

nginx: [emerg] bind() to [::]:80 failed (98: Address already in use)
Correct syntax:

server {
    listen 80;
    listen [::]:80 ipv6only=on;
}`
